README.md

```diff
@@ -34,6 +34,10 @@ If running for first time, do the setup as above and then run this script:
 ```sh
 ./start-nightfall -l | -g [-s]
 ```
+You can also start application with your own hosted geth. steps to follow
```

Comment:
When adding instructions, let us align to the already existing instructions. 
Please remove these bullet points and add the instructions in the paragraph below as a continuation to the prior instruction.

Reply:
done